src/util/native_fs_utils.js

```diff
@@ -289,6 +289,44 @@ function should_retry_link_unlink(err) {
     return should_retry_general || should_retry_gpfs || should_retry_posix;
 }
 
+/**
+ * stat_ignore_enoent unlinks a file and if recieved an ENOENT error it'll not fail
+ * @param {nb.NativeFSContext} fs_context
+ * @param {string} file_path
+ * @returns {Promise<nb.NativeFSStats>}
+ */
+async function stat_ignore_enoent(fs_context, file_path) {
+    try {
+        return await nb_native().fs.stat(fs_context, file_path);
+    } catch (err) {
+        if (err.code !== 'ENOENT') throw err;
+    }
+}
```

Comment:
I reviewed commit 4bf40d4bda9066ac17fb8da8088c73315edfaa73:
This function was not added/changed as a part of PR #8809.

Reply:
@shirady you are correct, this function was added but not used at all, I removed it, can you please re-review?